internal/device/device.go

```diff
@@ -0,0 +1,53 @@
+package device
+
+import (
+	"fmt"
+	"time"
+)
+
+func AmICharged() {
+	fmt.Println("Am I charged???")
+	fmt.Println()
+	powerStatus := GetPower()
+
+	percent, status := PowerStats(powerStatus)
+
+	if isCharged(status) {
```

Comment:
Only one of these can be true, can't they, so perhaps a `switch` statement would be better here. Something like:

```go
switch status {
case FullyCharged:
	...
case NearlyCharged:
	...
case Charging:
	...
}
```

Reply:
(where `FullyCharged`, etc, are constants that we've defined to make our code nice and readable, and independent of whatever the actual string values are)

Reply:
resolved in 1ccc772349311a8c9c50441562461c48885f8418